.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: false
 language: go
 go:
-  - 1.8.x
+  - 1.11.x
```

Comment:
@domgreen I had to update go version to make CI build happy. After updating dependencies with "dep ensure -update", `make all` was failing with `go 1.8`.

Reply:
Hey @domgreen, please let me know if we are fine with this change or you have any suggestions.